felix/bpf-gpl/tc.c

```diff
@@ -166,36 +166,6 @@ static CALI_BPF_INLINE int calico_tc(struct __sk_buff *skb)
 		case CALI_SKB_MARK_BYPASS_FWD:
```

Comment:
Maybe change to an `if` statement since there is no other branch in `switch`?

Reply:
I thought about that because (1) smaller diff (2) may be handy in the future and (3) compiler will figure it out :-)